dtbootstrap-anima-keyinfra.xml

```diff
@@ -1738,12 +1738,23 @@ locator3  = [O_IPv6_LOCATOR, fe80::1234, 41, nil]]]></artwork>
             </t>
             <t>
               Use of TLS 1.3 (or newer) is encouraged.
-              TLS 1.2 or newer is REQUIRED on the Pledge side.
-              TLS 1.3 (or newer) SHOULD be available on the Registrar server interface,
-              and the Registrar client interface, but TLS 1.2 MAY be used.
-              TLS 1.3 (or newer) SHOULD be available on the MASA server interface, but TLS
-              1.2 MAY be used.
+              TLS 1.2 or newer is REQUIRED on the pledge side.
+              TLS 1.3 (or newer) SHOULD be available on the registrar server
+              interface, and the registrar client interface, but TLS 1.2 MAY
+              be used.
+              When TLS 1.3 is used the use of Server Name Indicator (SNI,
+              <xref target="RFC6066" />) is not required, per <xref
+              target="RFC8446" /> section 9.2, this
+              specification is an application profile specification.
```

Comment:
```suggestion
              While the use of a Server Name Indicator (SNI) is not required per this (<xref target="RFC8446" />, Section 9.2) Application Profile specification, Pledges MAY include one. 
              Registrars MUST ignore any SNI that they see.
```

(This aligns with BRSKI-CLOUD needs)

Reply:
suggesting to change to "Server Name Indication (SNI)"  (i.e. use the official term) and to use lowercase on "application profile specification" and for "pledges".  (Conforms to RFC 8995 caps use)

We can also commit this suggestion soon to allow further wordsmithing.

Reply:
( I can't commit my own suggestion, it seems)